package.json

```diff
@@ -43,8 +44,7 @@
     "node": ">= 10"
   },
   "files": [
-    "dist/**/*.js",
-    "dist/**/*.ts"
+    "dist/"
```

Comment:
what files weren't match by the two includes?

Reply:
Felt it was more semantic to say that the `dist` folder is the package output in its entirety. Also `*.js.map` was missing so just using `dist/` covered everything.

Reply:
gotcha, the map files are what I was looking for--I think ok to use `dist/` but have been burnt in the past by adding large files by accident :)